README.md

```
@@ -1,461 +1,249 @@
+   REDISHOST=127.0.0.1
+   REDISPORT=6379
+   BROKER_URL=redis://127.0.0.1:6379/0
+   CELERY_QUEUE_NAME=dev
+
+   # Project & Repo Management
+   PROJECT_PATH=projects
+   ```
 
-👉 Install the Potpie Slack App: [Here](https://slack.potpie.ai/slack/install)
+   > **`CHAT_MODEL`** and **`INFERENCE_MODEL`** are used for agent reasoning and knowledge graph generation respectively. Model names follow the `provider/model_name` format as expected by [LiteLLM](https://docs.litellm.ai/docs/providers).
 
```

markdownlint-cli2 (0.20.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)